packages/dprint/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE=https://dprint.dev/
+TERMUX_PKG_DESCRIPTION="Pluggable and configurable code formatting platform written in Rust."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```

Comment:
This only has to be specified if the license file has a non-standard name or is in a non-standard location.
https://github.com/termux/termux-packages/blob/6525272eacd0c47bc6a9b0564e5688d99d3df1f5/scripts/build/termux_step_install_license.sh#L33-L96

Reply:
That's typical and is automatically handled by the license install step.

Reply:
i see. ✌️😁

Reply:
> https://github.com/termux/termux-packages/blob/6525272eacd0c47bc6a9b0564e5688d99d3df1f5/scripts/build/termux_step_install_license.sh#L33-L96

This can be a bit hard to read if you aren't used to Bash or our build system.
Basically we're looping through the licenses listed in `TERMUX_PKG_LICENSE`.
We then categorize licenses into two groups "specific" and "generic" licenses.

For "specific" licenses, meaning licenses that contain authorship information about the project, like the MIT license we then loop over the `COMMON_LICENSE_FILES` array,
and copy every found match into the package.

"Generic" licenses, without authorship information, are instead symlinked into the package from the `termux_licenses` package which contains the actual license texts.